### DISCUSSION TITLE
nvim-tree starts slowly on my wsl2

### DISCUSSION THREAD

**Discussion by ohwmy** [nvim-tree/nvim-tree.lua#1670] [Q&A]
My laptop CPU is Intel i7-12700H and RAM 16G. When I run neovim on wsl2 Ubuntu 20.04, it takes 270ms to start. But it runs faster(46ms) after I comment nvim-tree in `init.lua`.

So I'm curious if it's common or something goes wrong.

Here is my `init.lua`.
<img width="580" alt="image" src="https://user-images.githubusercontent.com/104249138/196329465-139b7e0b-4868-45a7-9cf6-fae950801675.png">

**Comment by alex-courtis** [comment]
If you are experiencing performance issues, please [enable profiling](https://github.com/nvim-tree/nvim-tree.lua#performance-issues) and collect the logs.

Please raise a [bug report](https://github.com/nvim-tree/nvim-tree.lua/issues/new?assignees=&labels=bug&template=bug_report.yml) if you don't see anything obvious in the logs.
